config/samples/portal/dcr.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: tyk.tyk.io/v1alpha1
+kind: ApiDefinition
+metadata:
+  name: httpbin
+  namespace: tyk
+spec:
+  name: httpbin
+  enable_jwt: true
+  protocol: http
+  active: true
+  proxy:
+    target_url: http://httpbin.org
+    listen_path: /httpbin
+    strip_listen_path: true
+  strip_auth_data: true
+  jwt_signing_method: rsa
+  jwt_source: aHR0cDovL2VjMi0xMy00MS0xNjUtMTUuZXUtd2VzdC0yLmNvbXB1dGUuYW1hem9uYXdzLmNvbTo4MTgwL2F1dGgvcmVhbG1zL21hc3Rlci9wcm90b2NvbC9vcGVuaWQtY29ubmVjdC9jZXJ0cw==
+  jwt_identity_base_field: sub
```

Comment:
I would use clientID here as opposed to sub. Usually you would want to rate-limit by the client, not the logged in user which the sub represents in OIDC.

Reply:
Fair point.  Will correct